### ISSUE TITLE
[Bug] swipe item is not accessible for voice over/accessibility inspector in collection view

### ISSUE THREAD

**Issue by shalinasanga** [xamarin/Xamarin.Forms#13861]
### Description
swipe item in collection view can not access (read) using accessibility inspector or voice over in iphone. it's accessible via parent Swipe View only. due to that cannot distinguish particular swipe item action.
**tested only in iOS**

### Steps to Reproduce

1. in CollectionView add Swipe View and add `AutomationProperties.IsInAccessibleTree="True"`
2. add one or two Swipe Item
3. in each Swipe Item add `AutomationProperties.IsInAccessibleTree="True"
                                               AutomationProperties.Name="some name"` to enable accessibility support
4. Run the app on simulator and swipe collection view item to make visible swipe item
5. use accessibility inspector to identify inside of swipe item
6. accessibility inspector can not identify Name property, same in device voice over

### Expected Behavior
Each swipe items must be uniquely identify using `AutomationProperties.Name`
### Actual Behavior
Can not identify swipe items uniquely 
### Basic Information

- Version with issue: 5.0.0.2012
- Last known good version: N/A
- Platform Target Frameworks: 
  - iOS:  
- NuGet Packages: xamarin forms
- Affected Devices: iOS 12 Simulator and iphone 6

### Environment

<details><summary>Show/Hide Visual Studio info</summary>

```
=== Visual Studio Community 2019 for Mac ===

Version 8.8.9 (build 10)
Installation UUID: 857463c0-a5cf-4373-ba10-ef57c4237d03
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 6.18.0.23 (d16-6 / 088c73638)

	Package version: 612000122

=== Mono Framework MDK ===

Runtime:
	Mono 6.12.0.122 (2020-02/c621c35ffa0) (64-bit)
	Package version: 612000122

=== Roslyn (Language Service) ===

3.8.0-5.20519.18+4c195c3ac1974edcefa76774d7a59a2350ec55fa

=== NuGet ===

Version: 5.8.0.6860

=== .NET Core SDK ===

SDK: /usr/local/share/dotnet/sdk/5.0.103/Sdks
SDK Versions:
	5.0.103
	5.0.102
	5.0.101
	5.0.100
	3.1.406
	3.1.405
	3.1.404
	3.1.403
	3.1.401
	3.1.302
	3.1.301
	3.1.300
	3.1.202
	3.1.200
	3.1.102
	3.1.101
	3.1.100
	3.0.101
	3.0.100
MSBuild SDKs: /Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/MSBuild/Current/bin/Sdks

=== .NET Core Runtime ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	5.0.3
	5.0.2
	5.0.1
	5.0.0
	3.1.12
	3.1.11
	3.1.10
	3.1.9
	3.1.7
	3.1.6
	3.1.5
	3.1.4
	3.1.2
	3.1.1
	3.1.0
	3.0.1
	3.0.0
	2.1.23
	2.1.21
	2.1.20
	2.1.19
	2.1.18
	2.1.17
	2.1.16
	2.1.15
	2.1.14
	2.1.13

=== .NET Core 3.1 SDK ===

SDK: 3.1.406

=== Xamarin.Profiler ===

Version: 1.6.12.29
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Updater ===

Version: 11

=== Apple Developer Tools ===

Xcode 12.4 (17801)
Build 12D4e

=== Xamarin.Mac ===

Version: 7.4.0.4 (Visual Studio Community)
Hash: 5a05865f6
Branch: xcode12.4
Build date: 2021-01-28 02:30:22-0500

=== Xamarin.iOS ===

Version: 14.10.0.4 (Visual Studio Community)
Hash: 5a05865f6
Branch: xcode12.4
Build date: 2021-01-28 02:30:23-0500

=== Xamarin Designer ===

Version: 16.8.0.510
Hash: 44e3f3ce9
Branch: remotes/origin/d16-8
Build date: 2020-12-10 00:06:14 UTC

=== Xamarin.Android ===

Version: 11.1.0.26 (Visual Studio Community)
Commit: xamarin-android/d16-8/a36ce73
Android SDK: /Users/agrando/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		None installed

SDK Tools Version: 26.1.1
SDK Platform Tools Version: 30.0.5
SDK Build Tools Version: 30.0.3

Build Information: 
Mono: 5e9cb6d
Java.Interop: xamarin/java.interop/d16-8@79d9533
ProGuard: Guardsquare/proguard/proguard6.2.2@ebe9000
SQLite: xamarin/sqlite/3.32.1@1a3276b
Xamarin.Android Tools: xamarin/xamarin-android-tools/d16-8@2fb1cbc

=== Microsoft OpenJDK for Mobile ===

Java SDK: /Users/agrando/Library/Developer/Xamarin/jdk/microsoft_dist_openjdk_1.8.0.25
1.8.0-25
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Android SDK Manager ===

Version: 16.8.0.32
Hash: 01a7774
Branch: remotes/origin/d16-8
Build date: 2021-01-14 00:34:58 UTC

=== Android Device Manager ===

Version: 16.8.0.46
Hash: 0a81419
Branch: remotes/origin/d16-8
Build date: 2021-01-14 00:35:22 UTC

=== Build Information ===

Release ID: 808090010
Git revision: 0061f975f58a5dba53e339f116371f4853edeb75
Build date: 2021-02-11 11:16:18-05
Build branch: release-8.8
Xamarin extensions: 0061f975f58a5dba53e339f116371f4853edeb75

=== Operating System ===

Mac OS X 10.16.0
Darwin 20.3.0 Darwin Kernel Version 20.3.0
    Thu Jan 21 00:07:06 PST 2021
    root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64


```

</details>

### Reproduction Link

[sample project](https://github.com/shalinasanga/Accessibility2)

### Workaround

if you have only one swipe item you can use swipe view to identify you swipe item action
`<SwipeView AutomationProperties.IsInAccessibleTree="True"
                                   AutomationProperties.Name="use this to explain your swipe action">
                            <SwipeView.RightItems >
                                <SwipeItems >
                                    <SwipeItem Text="d1" AutomationProperties.IsInAccessibleTree="True"
                                               AutomationProperties.Name="not working" BackgroundColor="Red"/>                                  
                                </SwipeItems>
                            </SwipeView.RightItems>
<Grid></Grid>
</SwipeView>`
